components/progressbar/metadata/meter.yml

```
@@ -1,34 +1,71 @@
 name: Meter
-description: Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+description: |
+  Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+  Meter should be only used in size `S` and `L` and could use the meter t-shirt sized classnames. [The Meter table of options](https://spectrum.adobe.com/page/meter/#Table-of-options) provides further details on which variants are available.
```

```suggestion
  Meter should be only used with `S` and `L`  t-shirt size classes along with Progress Bar t-shirt size classes. [The Meter table of options](https://spectrum.adobe.com/page/meter/#Table-of-options) provides further details on which variants are available.
```

This sentence wasn't clear to me. Here is a suggestion of what I think was intended.